tests/pass/SimpleOr.java

```diff
@@ -2,7 +2,11 @@
 
 public class SimpleOr {
     public static void main(String[] args) {
-        if (true || false){
+        if (true || true){
+            
+        }
+
+        if (true && true){
```

Comment:
Is it necessary to write conditionals to test boolean operations?

AFAIK, booleans exist independently of conditionals.

Maybe make the `SimpleOr` test simpler, like: `bool truth = false || true;`

Reply:
Also a bit strange that the `SimpleOr` test case tests the boolean and (`&&`) operation